lib/components/inputs/IpSubnetTextBox/IpSubnetTextBox.tsx

```diff
@@ -50,10 +51,11 @@ function IpSubnetTextBox(props: IpSubnetTextBoxProps) {
     isRequired,
     info,
     shouldConvertSubnet2Mask = true,
+    fixedSubnet,
     ...rest
   } = props
   const { disabled } = rest
-  const [ipVal, subnet] = value ? value.split('/') : ['...', '']
+  const [ipVal, subnet] = value ? value.split('/') : ['...', fixedSubnet ?? '']
```

Comment:
smth small (which I also forgot to mention): we have `EMPTY_STRING` variable, so better use it instead of `''`(I see that it wasn't used before, but still:)

Reply:
Fixed, if we'll have time, we can refactor all of this component(s) :)